drafts/draft-ietf-ippm-ioam-ipv6-options.xml

```diff
@@ -146,20 +126,20 @@
       networks, such as the IPv6 Performance and Diagnostic Metrics
       Destination Option described in <xref target="RFC8250"/>.</t>
```

Comment:
IMHO we could add a single sentence here that states something about IOAM implementations in IPv6.
That way we acknowledge the ask to provide for a stronger justification for the need of new v6 HbyH options.
E.g., we could add a sentence like "At the time this document was written, several implementations of IOAM for IPv6 exist, e.g., IOAM for IPv6 in the Linux Kernel (supported from Kernel version 5.15 onwards [reference]) or IOAM for IPv6 in VPP [fd.io]."

Reply:
Done!